am-lib/src/test/java/integration/uk/gov/hmcts/reform/amlib/GrantAccessIntegrationTest.java

```diff
@@ -111,4 +111,12 @@ private ExplicitAccessGrant createPermissionsForResourceForMultipleUsers(String
             .relationship(roleName)
             .build();
     }
+
+    @Test
+    void whenCreatingResourceAccessResourceAccessAppearsInDatabaseWIthNullRelationship() {
```

Comment:
the name of the method does not make sense

Moreover, it is not clear what this test case is trying to achieve?

Reply:
It's more specific to check null relationship/role. And check  Explicit Access record gets created or not with null relationship.

Reply:
will rename this